haas/api.py

```diff
@@ -133,7 +133,65 @@ def project_detach_node(project, node):
     project.nodes.remove(node)
     db.session.commit()
 
-
+@rest_call('PUT', '/network/<network>/access/<project>')
```

Comment:
Could you add a `Schema` argument for this call and the next, similar to this:
```python
@rest_call('PUT', '/project/<project>', Schema({'project': basestring}))
def project_create(project):
```

Now that we've merged #596, all calls need a `Schema`.